packages/xpetra/src/CrsMatrix/Xpetra_TpetraCrsMatrix_decl.hpp

```diff
@@ -224,8 +171,7 @@ namespace Xpetra {
     TpetraCrsMatrix (const Teuchos::RCP<const Map<LocalOrdinal,GlobalOrdinal,Node> >& rowMap,
         const Teuchos::RCP<const Map<LocalOrdinal,GlobalOrdinal,Node> >& colMap,
         const local_matrix_type& lclMatrix,
-        const Teuchos::RCP<Teuchos::ParameterList>& params = null)
-    : mtx_(Teuchos::rcp(new Tpetra::CrsMatrix<Scalar, LocalOrdinal, GlobalOrdinal, Node>(toTpetra(rowMap), toTpetra(colMap), lclMatrix, params))) {  }
+        const Teuchos::RCP<Teuchos::ParameterList>& params = null);
```

Comment:
This is a great opportunity to purge header file includes (e.g., for `toTpetra`) from the decl header.